src/celeritas/ext/detail/GeantExceptionHandler.cc

```diff
@@ -42,7 +42,7 @@ G4bool GeantExceptionHandler::Notify(char const* origin_of_exception,
             throw err;
         case JustWarning:
             // Display a message
-            CELER_LOG(warning) << err.what();
+            CELER_LOG(error) << err.what();
```

Comment:
What's the reasoning behind this change?

Reply:
The only times I've seen this condition from Geant4 is when something is definitely wrong (an error) but it's recoverable. (Although sometimes it means the code is about to crash, too.) That's the equivalent logger level for us: warning is like "probably wrong, hope you know what you're doing"; error is "this is wrong but we can keep going".